msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md

```diff
@@ -1314,10 +1314,10 @@ The following table provides the error codes:
 | **404** | Meeting not found or not started. If you receive this error, ensure that you start the meeting and select start captions. After captions are enabled in the meeting, you can begin POSTing captions into the meeting.|
 | **500** |Internal server error. For more information, [contact support or provide feedback](../feedback.md).|
 
```

Comment:
Missing title to be added linked to the item in my previous comment:
Receive real-time Teams meeting events

Reply:
Linked accordingly and updated.